Extension/src/LanguageServer/client.ts

```diff
@@ -1303,91 +1319,8 @@ export class DefaultClient implements Client {
-                            this.semanticTokensProviderDisposable = vscode.languages.registerDocumentSemanticTokensProvider(util.documentSelector, this.semanticTokensProvider, semanticTokensLegend);
-                        }
-                        // Listen for messages from the language server.
-                        this.registerNotifications();
-                    }
-                    // update all client configurations
-                    this.configuration.setupConfigurations();
-                    initializedClientCount++;
-                    // count number of clients, once all clients are configured, check for trusted compiler to display notification to user and add a short delay to account for config provider logic to finish
-                    if ((vscode.workspace.workspaceFolders === undefined) || (initializedClientCount >= vscode.workspace.workspaceFolders.length)) {
-                        // Timeout waiting for compile_commands.json and config providers.
-                        // The quick pick options will update if they're added later on.
-                        clients.forEach(client => {
-                            if (client instanceof DefaultClient) {
-                                global.setTimeout(() => {
-                                    client.configStateReceived.timeout = true;
-                                    void client.handleConfigStatusOrPrompt();
-                                }, 15000);
-                            }
-                        });
-                        // The configurations will not be sent to the language server until the default include paths and frameworks have been set.
-                        // The event handlers must be set before this happens.
-                        compilerDefaults = await this.requestCompiler();
-                        DefaultClient.updateClientConfigurations();
-                        clients.forEach(client => {
-                            if (client instanceof DefaultClient) {
-                                client.configStateReceived.compilers = true;
-                                void client.handleConfigStatusOrPrompt();
-                            }
-                        });
-                    }
-                } catch (err) {
-                    this.isSupported = false;   // Running on an OS we don't support yet.
-                    if (!failureMessageShown) {
-                        failureMessageShown = true;
-                        void vscode.window.showErrorMessage(localize("unable.to.start", "Unable to start the C/C++ language server. IntelliSense features will be disabled. Error: {0}", String(err)));
-                    }
-                }
-            });
         } catch (errJS) {
             const err: NodeJS.ErrnoException = errJS as NodeJS.ErrnoException;
             this.isSupported = false;   // Running on an OS we don't support yet.
```

Comment:
There's extra white space before the comment here. I thought we previously had a linter rule for that.